nx.json

```diff
@@ -2,9 +2,14 @@
   "$schema": "./node_modules/nx/schemas/nx-schema.json",
   "extends": "nx/presets/npm.json",
   "targetDefaults": {
+    "copy-license": {
+      "cache": true,
+      "outputs": ["{projectRoot}/LICENSE"]
+    },
```

Comment:
Is this required?

Reply:
It just copies `LICENSE` file to the package before bundling.

Reply:
What I wonder is why other packages, e.g. Material UI, don't need this process. I feel all of them should have the same license?

Reply:
They also copy `LICENSE` file but that happens through a separate script which also copies a bunch of other stuff that is not needed in zero related packages. So I created a script just to copy the `LICENSE`

Reply:
I see no problem with this.

Reply:
Alright, awesome.